lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py

```diff
@@ -241,7 +244,7 @@ def cacheCommitUi(parent, selectedFile):
 
     # The following call will set _cacheExportOptions.  Initial settings not
     # accessed on "query", is therefore an empty string.
-    mel.eval('mayaUsdTranslatorExport("fileOptionsScroll", "query", "", "mayaUsdCacheMayaReference_setCacheOptions")')
+    mel.eval('mayaUsdTranslatorExport("fileOptionsScroll", "query={exportOpts}", "", "mayaUsdCacheMayaReference_setCacheOptions")'.format(exportOpts=kTranslatorExportOptions))
```

Comment:
Need the same option string for the query so you don't get back those options (like parent scope).